vms/platformvm/config/configtest/configtest.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package configtest
+
+import (
+	"time"
+
+	"github.com/ava-labs/avalanchego/utils/units"
+)
+
+// Shared Unit test setup utilities for a platform vm packages
+
+var (
```

Comment:
in next PRs we'll consolidate a bunch of test objects creation here (config, context, clock).
For the time being we just repackage a few config parameters which are used in genesis, but not only in genesis creation